exasol_integration_test_docker_environment/test/test_hash_temp_dir.py

```diff
@@ -138,8 +148,8 @@ def test_directory_relative_paths_equal(self):
                                          use_relative_paths=True,
                                          hash_directory_names=False,
                                          hash_file_names=False)
-        hash1 = hasher.hash([self.test_dir1])
-        hash2 = hasher.hash([self.test_dir2])
+        hash1 = hasher.hash([simple_path_mapping(self.test_dir1)])
```

Comment:
Name of the test to similar to test_directory_with_relative_paths_fixed_hash(self)

Reply:
I removed this test now